### ISSUE TITLE
Make your own consoles backwards compatible.

### ISSUE THREAD

**Issue by Zebbidoo** [binogure-studio/city-game-studio-i18n#811]
I wish there was a feature when you develop a new console, where you can make it so that games from a previous console also apply to that console. A console having no games obviously makes it sell worse, but being backwards compatible should then take older games into account. Just like in the real world there should be a development/production cost associated with this.

Maybe the bonus to sales from old games should count less per game than new games, or maybe the bonus should wear off after a while? As it would in reality I assume.

A real world example would be developing something such as the gameboy color. It was greatly helped by the fact it could slow down it's processor to play all existing Gameboy games. Another would of course be the PS2, being able to play all PS1 games.

So in short, I wish my released console could be made backwards compatible with one or more of my old consoles, therefore helping it sell.

**Comment by SickoonSeriesX** [comment]
Great Idea

**Comment by xsellier** [comment]
Closing this one in favor of #659
